### ISSUE TITLE
Algorithm description points to a hash reference

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#692]
What is reported here is built on the develop branch.

The message string variable "algorithm_description" is interpreted as a hash reference instead of a scalar value.

```
# zonemaster-cli xn--ldaa.fr  --level INFO --show_module --test=DNSSEC --no-ipv6
Seconds Level     Module       Message
======= ========= ============ =======
   0.00 INFO      SYSTEM       Using version v3.0.3 of the Zonemaster engine.
(...)
   7.13 INFO      DNSSEC       All the tested RRset (SOA/DNSKEY/NS) are signed by each algorithm present in the DNSKEY RRset
   7.14 WARNING   DNSSEC       DNSKEY with tag 11102 and using algorithm 7 (HASH(0x8045cf660)) has a size (1024) smaller than the recommended one (2048).
   7.15 WARNING   DNSSEC       DNSKEY with tag 23400 and using algorithm 7 (HASH(0x8045cf660)) has a size (1024) smaller than the recommended one (2048).
   7.15 WARNING   DNSSEC       DNSKEY with tag 11102 and using algorithm 7 (HASH(0x8045cf660)) has a size (1024) smaller than the recommended one (2048).
   7.15 WARNING   DNSSEC       DNSKEY with tag 23400 and using algorithm 7 (HASH(0x8045cf660)) has a size (1024) smaller than the recommended one (2048).
```

The same thing, but with "raw" output.

```
# zonemaster-cli xn--ldaa.fr  --level INFO --show_module --test=DNSSEC --no-ipv6 --raw
   0.00 INFO      SYSTEM       SYSTEM:GLOBAL_VERSION version=v3.0.3
(...)
   5.93 INFO      DNSSEC       DNSSEC:ALL_ALGO_SIGNED 
   5.93 WARNING   DNSSEC       DNSSEC:DNSKEY_SMALLER_THAN_REC algorithm_description={"description":"RSASHA1-NSEC3-SHA1","mnemonic":"RSASHA1-NSEC3-SHA1","sig":1,"status":9}; algorithm_number=7; keysize=1024; keysizemax=4096; keysizemin=512; keysizerec=2048; keytag=23400
   5.94 WARNING   DNSSEC       DNSSEC:DNSKEY_SMALLER_THAN_REC algorithm_description={"description":"RSASHA1-NSEC3-SHA1","mnemonic":"RSASHA1-NSEC3-SHA1","sig":1,"status":9}; algorithm_number=7; keysize=1024; keysizemax=4096; keysizemin=512; keysizerec=2048; keytag=11102
   5.94 WARNING   DNSSEC       DNSSEC:DNSKEY_SMALLER_THAN_REC algorithm_description={"description":"RSASHA1-NSEC3-SHA1","mnemonic":"RSASHA1-NSEC3-SHA1","sig":1,"status":9}; algorithm_number=7; keysize=1024; keysizemax=4096; keysizemin=512; keysizerec=2048; keytag=23400
   5.94 WARNING   DNSSEC       DNSSEC:DNSKEY_SMALLER_THAN_REC algorithm_description={"description":"RSASHA1-NSEC3-SHA1","mnemonic":"RSASHA1-NSEC3-SHA1","sig":1,"status":9}; algorithm_number=7; keysize=1024; keysizemax=4096; keysizemin=512; keysizerec=2048; keytag=11102
```

**Comment by vlevigneron** [comment]
I'm on it

**Comment by matsduf** [comment]
Resolved by #693.
